packages/compass-smoke-tests/src/cli.ts

```diff
@@ -196,6 +252,33 @@ async function run() {
     console.log('Cleaning up sandbox');
     fs.rmSync(context.sandboxPath, { recursive: true });
   }
+
+  console.log('update from latest release to this package');
+  const releasepath = await getLatestRelease(
+    buildInfo.channel,
+    context.arch,
+    kind,
+    context.forceDownload
+  );
+
+  try {
+    const appName = buildInfo.productName;
+
+    const { appPath, uninstall } = install({
+      appName,
+      filepath: releasepath,
+      destinationPath: context.sandboxPath,
+    });
+
+    try {
+      runTest({ appName, appPath });
```

Comment:
Again: This should actually see if you can update from this version. I'm just including it as a kind of sanity check that releasepath works.